sdk/core/core-util/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../../../tsconfig.package",
   "compilerOptions": {
     "outDir": "./dist-esm",
-    "declarationDir": "./types/latest"
+    "declarationDir": "./types/latest",
+    "lib": ["dom"]
```

Comment:
can we shim out the surface we use instead of dragging in the whole `dom` typing? Especially for `core-util` which of all our packages shouldn't presume a DOM exists.

Reply:
Removed